src/types/annotation.ts

```diff
@@ -25,3 +25,5 @@ export interface DeleteAnnotation {
   end: string
   stickers?: any
 }
+
+export type StateAnnotations = {[stream: string]: Annotation[]}
```

Comment:
I'm really not in love with this name. Up for suggestions

Reply:
AnnotationList

that is what it is.  
plus the AnnotationState / StateAnnotations in the reducers is confusing.

Reply:
it's not quite a `list` though - it's a hashmap keyed on stream. It's kind of like a list, but kind of not